htdocs/compta/facture/class/facture.class.php

```diff
@@ -946,7 +946,7 @@ function createFromOrder($object)
 		$this->date = dol_now();
 		$this->source = 0;
 
-		$num=count($object->lines);
+		$num = (is_null($object->lines) ? 0 : count($object->lines)); // for avoid phpunit error : "Parameter must be an array or an object that implements Countable"
```

Comment:
@eldy 
I do not know, it was a mistake I had seen in Travis (php nightly build), you can remove this correction if you want.